sunpy/visualization/tests/test_draw.py

```diff
@@ -0,0 +1,20 @@
+"""
+Test draw
+"""
+import pytest
+
+from sunpy.tests.helpers import figure_test
+from sunpy.visualization import draw
+from sunpy.map.tests.conftest import aia171_test_map
+
+
+@figure_test
```

Comment:
You will need to add hashes to the figure hashes file for these to work correctly.

If you see https://app.circleci.com/pipelines/github/sunpy/sunpy/18331/workflows/27b6a082-24ed-4fc2-ad0e-7e06599b0594/jobs/79348/parallel-runs/0/steps/0-107

The artefacts with the hashes are there too.